### ISSUE TITLE
Typo in core/howto/process leads to misrendering

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7642]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7642|
|Type|enhancement|
|Created|2014-09-16 16:33:29Z|
|Branch|https://github.com/twisted/twisted/tree/process-howto-typo-7642|

There's a missing space.

Attachments:

* [t7642.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/73d/73dc8f5748a14c821e64febc5e461a606f6bf7b1/f074f02c3cc687b9c0d4b8d110dab8a189a777f2.diff) (702 bytes) - added by Alex on 2014-09-16 16:33:57Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7642 7642
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_process_howto_typo_7642 branches/process-howto-typo-7642
branch_author__khorn khorn
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1410885209020703 1410885209020703
changetime__1411101352094314 1411101352094314
version__None None
owner__khorn khorn

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Alex set owner to @glyph|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @khorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_3"></a>|@khorn commented|
|-|-|

(In [43138]) Branching to process-howto-typo-7642.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_4"></a>|@khorn commented|
|-|-|

(In [43139]) Fix typo in process howto. Refs [#7642](https://github.com/twisted/twisted/issues/7642)

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_5"></a>|@khorn commented|
|-|-|

Looks fine. Merging.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_6"></a>|@khorn set status to `closed`|
|-|-|

(In [43140]) Merge process-howto-typo-7642: Fix typo in process howto.

Author: Alex
Reviewer: khorn
Fixes: [#7642](https://github.com/twisted/twisted/issues/7642)

This fixes a minor typo in the process howto, which was causing a link to be
rendered incorrectly by Sphinx.
